cypress/support/commands.js

```diff
@@ -1,16 +1,13 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
 // -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
+Cypress.Commands.add('logoClick', () => { 
```

Comment:
Tests work and look ok, but I would avoid creating custom commands for trivial things like click. 
Plus, the name "logoClick" is not such a good description of the command since the command visits the homepage and then clicks. 
Similar with verifyUrlRedirect, the "redirect" word is mostly used when some link has been changed so instead of opening link A, it redirects you to link B ("When a web browser attempts to open a URL that has been redirected, a page with a different URL is opened.")

Reply:
Makes sense - I'll make the changes and push today